### ISSUE TITLE
When selecting IDP and role at the CLI, you still get prompted

### ISSUE THREAD

**Issue by ctennis** [okta/okta-aws-cli#24]
Would be nice if we specify these options at the command line, it just uses them as opposed to make you pick them from the picker. This would eliminate the need for using arrow keys and return when the command line is specified.

Similarly, in the case where there's only one option for each of these items, it should auto select as opposed to making you pick the one option from the list.

**Comment by nickgamb** [comment]
Agreed and this is planned. Expect to see support for multiple AWS environments and profiles soon.

**Comment by monde** [comment]
[GA v0.1.0](https://github.com/okta/okta-aws-cli/releases/tag/v0.1.0) has been released 🎉
